src/store.ts

```diff
@@ -6,11 +6,16 @@ import {
   Store,
   Reducer,
 } from "redux";
-import catalogReducers from "@thepalaceproject/web-opds-client/lib/reducers";
+
+import thunk from "redux-thunk";
+import catalogReducers from "@thepalaceproject/web-opds-client/lib/reducers/index";
 import { State as CatalogState } from "@thepalaceproject/web-opds-client/lib/state";
 import editorReducers, { State as EditorState } from "./reducers/index";
 
-import thunk from "redux-thunk";
+export interface CombinedState {
+  editor: EditorState;
+  catalog: CatalogState;
+}
```

Comment:
Any reason to have both `CombinedState` and `RootState`?

Reply:
No. No good reason, anyway. 😅